### ISSUE TITLE
Dropdown Warning: findDOMNode is deprecated in StrictMode

### ISSUE THREAD

**Issue by anselal** [ant-design/ant-design#41183]
### Reproduction link

[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/antd-reproduction-template-forked-3d5j7d?file=/index.js)

### Steps to reproduce

Click on the dropdown

### What is expected?

No warning

### What is actually happening?

Warning: findDOMNode is deprecated in StrictMode. findDOMNode was passed an instance of DomWrapper4 which is inside StrictMode. Instead, add a ref directly to the element you want to reference

| Environment | Info |
| --- | --- |
| antd | 5.3.0 |
| React | 18.2 |
| System | Ubuntu 20.04 |
| Browser | Firefox 110.0.1 |

**Comment by Yuiai01** [comment]
Maybe similar to this problem https://github.com/ant-design/ant-design/issues/40340, let me try to fix it ~

**Comment by anselal** [comment]
Yeah, for some reason the link doesn't render anything. Will create a new one

**Comment by anselal** [comment]
> Duplicate of #35361

Issue #35361 is about antd v4.
Mine is about antd v5.

**Comment by AmirovQodir** [comment]
Same problem with same versions
`"antd": "^5.3.0",
 "react": "^18.2.0",`

**Comment by AmirovQodir** [comment]
I think this issue is not actually duplicated!

**Comment by KevinYouu** [comment]
Same problem 
`"antd": "^5.3.1", "react": "^18.2.0"`

**Comment by jglee96** [comment]
We can still reproduce it in Ant Design Dropdown demo page code. - [Arrow pointing at the center](https://ant.design/components/dropdown#components-dropdown-demo-arrow-center)

**Comment by soadergi** [comment]
You can wrap children in a tag, a div for example. This did help me to stop findDOMNode in console

**Comment by papazos** [comment]
the warning will be automatically **disappeared in production** so that's fine by me.
